executor/runtime/docker/docker_config.go

```diff
@@ -166,6 +166,22 @@ func shouldStartSpectatord(cfg *config.Config, c runtimeTypes.Container) bool {
 	return true
 }
 
+func shouldStartAtlasd(cfg *config.Config, c runtimeTypes.Container) bool {
+	enabled := cfg.ContainerAtlasd
+	if !enabled {
+		return false
+	}
+
+	if cfg.AtlasdServiceImage == "" {
+		return false
+	}
+	return true
+}
+
+func shouldStartAtlasAgent(cfg *config.Config, c runtimeTypes.Container) bool {
+	return !shouldStartAtlasd(cfg, c)
```

Comment:
Is this because Atlast-agent and atlasd are incompatible with each other? (one in the container, one on the host?)
if so can you add a comment here to say that?